.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: "1.19.1"
+          go-version: "1.20.1"
```

Comment:
Is this an action bug?

Reply:
No. For `golangci-lint`, we'd like to use a higher version of golang thus to allow it to report more warnings